export-coin-list.user.js

```diff
@@ -29,7 +29,7 @@
     const listItem = jQuery('<li></li>');
     listItem.append(button);
 
-    jQuery('.page-title-box .dropdown-menu').append(listItem);
+    jQuery('.btn-group.pull-right .dropdown-menu').append(listItem);
```

Comment:
I would prefer making this a bit more specific to the action box instead of just the button group to prevent selecting any other button groups they might add. However slim the chance they would also have it pulled right.

i.e.: '.c-page-heading__actions .btn-group .dropdown-menu'